### ISSUE TITLE
collection "zone d'intérêt"

### ISSUE THREAD

**Issue by brunob** [geodiversite/geodiversite#9]
Permettre de créer des collection à partir d'un tracé (ou import kml/gpx). Ce type de collection est "statique", on ne peut pas y ajouter de média. La liste des médias est générée à partir du résultat d'un "point in polygon" mysql.

**Comment by ubiquando** [comment]
dans mon idée, ce serait plutôt une collection "dynamique" ou "virtuelle", dans le sens qu'on ne connaît son contenu qu'à l'instant T, mais on ne peut pas ajouter/enlever explicitement de media à la collection.
Pas sûr que ce soit vraiment simple...

**Comment by brunob** [comment]
Oui, c'est exactement ce que disais plus haut, on ne peut pas ajouter de media **manuellement**, mais tout media qui entre dans la zone géographique de la collection en fait partie automatiquement.

**Comment by ubiquando** [comment]
la fonction magique 'geometrie_dans_polygone' de gisgeom nous aide beaucoup pour cela !
Personnellement je l'utilise pour des catégories, mais avec cela les collections "zone d'intérêt" sont vraiment possibles.

`  Dans une boucle GIS : 
#SET{poly,#GEOMETRY|zoner_geometrie{wkt,0.05}}

Puis :

<BOUCLE_points_proches(GIS){type=Point}{geometrie_dans_polygone #GET{poly}}> 
`

**Comment by brunob** [comment]
Vi, ya des pépites dans gis_geo cf https://git.spip.net/spip-contrib-extensions/gis_geometries/commit/5aa35d8e5ccb2194ab45a9e22c89b8a20559fc7c

Ça vaudrait le coup de compléter la doc de ce plugin.
